src/components/textToSpeech/js/TextToSpeech.js

```diff
@@ -72,9 +72,11 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
             onSpeechQueued: null
         },
         members: {
-            queue: []
+            queue: [],
+            //
+            currentUtterance: {}
```

Comment:
The `queue` member contains the queue of texts yet to be spoken; it has nothing to do with the queueing behaviour I implemented, which (as you state above) is probably more correctly described as a throttle to prevent more pause/resume commands from being sent via the wrapper while outstanding pause/resume requests are still resolving.